### ISSUE TITLE
Error running analyze.py

### ISSUE THREAD

**Issue by aweaver1fandm** [kahst/BirdNET-Analyzer#558]
**Describe the bug**
The analyzer causes a core dump.  I git pulled the repo today 1/28/2025

**To Reproduce**
python -m birdnet_analyzer.analyze
Illegal instruction (core dumped)

**Expected behavior**
Not a core dump

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version: 24.04

**Additional context**
Created a conda environment.  List of installed packages in the environment below.  ffmpeg is installed on the system as well.
For birdnet I did: pip install birdnet[and-cuda]
I also did the birdnet install without cuda with same result

packages in environment at /opt/miniconda3/envs/birdnet:

Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
absl-py                   2.1.0                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
audioread                 3.0.1                    pypi_0    pypi
birdnet                   0.1.6                    pypi_0    pypi
birdnetlib                0.18.0                   pypi_0    pypi
bzip2                     1.0.8                h5eee18b_6  
ca-certificates           2024.12.31           h06a4308_0  
cachetools                5.5.1                    pypi_0    pypi
certifi                   2024.12.14               pypi_0    pypi
cffi                      1.17.1                   pypi_0    pypi
charset-normalizer        3.4.1                    pypi_0    pypi
contourpy                 1.3.1                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
flatbuffers               25.1.24                  pypi_0    pypi
fonttools                 4.55.7                   pypi_0    pypi
gast                      0.6.0                    pypi_0    pypi
google-auth               2.38.0                   pypi_0    pypi
google-auth-oauthlib      1.2.1                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.70.0                   pypi_0    pypi
h5py                      3.12.1                   pypi_0    pypi
idna                      3.10                     pypi_0    pypi
joblib                    1.4.2                    pypi_0    pypi
keras                     2.15.0                   pypi_0    pypi
keras-tuner               1.4.7                    pypi_0    pypi
kiwisolver                1.4.8                    pypi_0    pypi
kt-legacy                 1.0.5                    pypi_0    pypi
lazy-loader               0.4                      pypi_0    pypi
ld_impl_linux-64          2.40                 h12ee557_0  
libclang                  18.1.1                   pypi_0    pypi
libffi                    3.4.4                h6a678d5_1  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
librosa                   0.10.2.post1             pypi_0    pypi
libstdcxx-ng              11.2.0               h1234567_1  
libuuid                   1.41.5               h5eee18b_0  
llvmlite                  0.44.0                   pypi_0    pypi
markdown                  3.7                      pypi_0    pypi
markupsafe                3.0.2                    pypi_0    pypi
matplotlib                3.10.0                   pypi_0    pypi
ml-dtypes                 0.3.2                    pypi_0    pypi
msgpack                   1.1.0                    pypi_0    pypi
ncurses                   6.4                  h6a678d5_0  
numba                     0.61.0                   pypi_0    pypi
numpy                     1.26.4                   pypi_0    pypi
nvidia-cublas-cu12        12.2.5.6                 pypi_0    pypi
nvidia-cuda-cupti-cu12    12.2.142                 pypi_0    pypi
nvidia-cuda-nvcc-cu12     12.2.140                 pypi_0    pypi
nvidia-cuda-nvrtc-cu12    12.2.140                 pypi_0    pypi
nvidia-cuda-runtime-cu12  12.2.140                 pypi_0    pypi
nvidia-cudnn-cu12         8.9.4.25                 pypi_0    pypi
nvidia-cufft-cu12         11.0.8.103               pypi_0    pypi
nvidia-curand-cu12        10.3.3.141               pypi_0    pypi
nvidia-cusolver-cu12      11.5.2.141               pypi_0    pypi
nvidia-cusparse-cu12      12.1.2.141               pypi_0    pypi
nvidia-nccl-cu12          2.16.5                   pypi_0    pypi
nvidia-nvjitlink-cu12     12.2.140                 pypi_0    pypi
oauthlib                  3.2.2                    pypi_0    pypi
openssl                   3.0.15               h5eee18b_0  
opt-einsum                3.4.0                    pypi_0    pypi
ordered-set               4.1.0                    pypi_0    pypi
packaging                 24.2                     pypi_0    pypi
pillow                    11.1.0                   pypi_0    pypi
pip                       24.2            py310h06a4308_0  
platformdirs              4.3.6                    pypi_0    pypi
pooch                     1.8.2                    pypi_0    pypi
protobuf                  4.25.6                   pypi_0    pypi
pyasn1                    0.6.1                    pypi_0    pypi
pyasn1-modules            0.4.1                    pypi_0    pypi
pycparser                 2.22                     pypi_0    pypi
pydub                     0.25.1                   pypi_0    pypi
pyparsing                 3.2.1                    pypi_0    pypi
python                    3.10.16              he870216_1  
python-dateutil           2.9.0.post0              pypi_0    pypi
readline                  8.2                  h5eee18b_0  
requests                  2.32.3                   pypi_0    pypi
requests-oauthlib         2.0.0                    pypi_0    pypi
resampy                   0.4.3                    pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
scikit-learn              1.6.1                    pypi_0    pypi
scipy                     1.15.1                   pypi_0    pypi
setuptools                75.1.0          py310h06a4308_0  
six                       1.17.0                   pypi_0    pypi
soundfile                 0.13.1                   pypi_0    pypi
soxr                      0.5.0.post1              pypi_0    pypi
sqlite                    3.45.3               h5eee18b_0  
tensorboard               2.15.2                   pypi_0    pypi
tensorboard-data-server   0.7.2                    pypi_0    pypi
tensorflow                2.15.1                   pypi_0    pypi
tensorflow-estimator      2.15.0                   pypi_0    pypi
tensorflow-io-gcs-filesystem 0.37.1                   pypi_0    pypi
termcolor                 2.5.0                    pypi_0    pypi
threadpoolctl             3.5.0                    pypi_0    pypi
tk                        8.6.14               h39e8969_0  
tqdm                      4.67.1                   pypi_0    pypi
typing-extensions         4.12.2                   pypi_0    pypi
tzdata                    2025a                h04d1e81_0  
urllib3                   2.3.0                    pypi_0    pypi
watchdog                  2.1.9                    pypi_0    pypi
werkzeug                  3.1.3                    pypi_0    pypi
wheel                     0.44.0          py310h06a4308_0  
wrapt                     1.14.1                   pypi_0    pypi
xz                        5.4.6                h5eee18b_1  
zlib                      1.2.13               h5eee18b_1

**Comment by aweaver1fandm** [comment]
After a bit more testing, the issue appears to be with model.py.  I activated my conda environment and started python and imported model here is the result

Python 3.11.0 (main, Mar  1 2023, 18:26:19) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import birdnet_analyzer.model as model
Illegal instruction (core dumped)

If I do the same with like audio it is fine:
Python 3.11.0 (main, Mar  1 2023, 18:26:19) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import birdnet_analyzer.audio as audio

**Comment by aweaver1fandm** [comment]
It does not look like my CPU has avx2 support.  Nothing shows up when I cat /proc/cpuinfo.  This is a virtual machine setup in vsphere so I may have to look at the settings to see how that gets passed through from the host to the virtual machine

**Comment by aweaver1fandm** [comment]
Enabled avx pass through/support for the virtual machine and then analyze ran fine.  Closing issue
